docs/plugins/router.md

```diff
@@ -31,9 +31,14 @@ Add the `NgxsRouterPluginModule` plugin to your root app module:
 ```ts
 import { NgxsModule } from '@ngxs/store';
 import { NgxsRouterPluginModule } from '@ngxs/router-plugin';
+import { AppRoutingModule } from './app-routing.module';
 
 @NgModule({
-  imports: [NgxsModule.forRoot([]), NgxsRouterPluginModule.forRoot()]
+  imports: [
+    AppRoutingModule, // 1. Init RouterModule.forRoot([ROUTES])
+    NgxsModule.forRoot([AppState]), // 2. Init NGXS Store with states
+    NgxsRouterPluginModule.forRoot() // 3. Init NGXS Router plugin
```

Comment:
With the `1.`, `2.` and `3.` are we saying that this order matters? If so that we should probably be explicit about it.
Otherwise the numbers are not necessary,

Reply:
Yes order matters

Reply:
Ok, please could you add a line above the code snippet stating that the order matters. ie. what needs to be before what.

Do you know why it is order sensitive?

Reply:
Because then you can get the router status incorrectly in NGXS Router plugin

Reply:
So, is it important to have the Router plugin after the Routing module for this reason. But it should be after the `NgxsModule.forRoot` for a different reason?

It would be great if we could remove these strange order dependencies somehow though.

Reply:
@splincode If the order matters, why did you now remove the details about the order that they must be in?